mlflow/utils/__init__.py

```diff
@@ -296,3 +296,7 @@ def __getattr__(self, attr):
         if isinstance(value, dict):
             return AttrDict(value)
         return value
+
+
+def get_parent_module(_module):
+    return _module[0: _module.rindex(".")]
```

Comment:
```suggestion
def get_parent_module(module):
    return module[0: module.rindex(".")]
```

or `mod`.

Reply:
```suggestion
def get_parent_module(_module):
    return _module[: _module.rindex(".")]
```

0 can be removed.

Reply:
Any chance that `.` is not found?

Reply:
The `_CaptureImportedModules` implementation ensures logging full module name, and this is covered by `tests/pyfunc/test_inferred_data_path.py` (see file `tests/pyfunc/custom_model/mod1/__init__.py` )